app/addons/auth/components.react.jsx

```diff
@@ -219,17 +218,17 @@ define([
     render: function () {
       return (
         <div className="auth-page">
-          <h3>Create Admins</h3>
+          <h3>Create User Accounts with ADMIN Privileges</h3>
 
           <p>
-            Before a server admin is configured, all clients have admin privileges. This is fine when
-            HTTP access is restricted to trusted users. <strong>If end-users will be accessing this
-            CouchDB, you must create an admin account to prevent accidental (or malicious) data
-            loss.</strong>
+            Before an admin user account is created for this CouchDB server instance,
+            all clients have ADMIN privileges.  <strong>If clients are not trusted users,
+            you must create an admin account to prevent accidental or malicious data loss.</strong>
           </p>
           <p>
-            Server admins can create and destroy databases, install and update _design documents, run
-            the test suite, and edit all aspects of CouchDB configuration.
+            Users with ADMIN privileges can create, modify, and delete databases and design documents;
+            run the test suite; and modify the CouchDB configuration. Users without ADMIN privileges
+            have read/write access to all databases. CouchDB can be configured to block access by anonymous users.
```

Comment:
I think such sentence requires a link to somewhere to answer on upcoming question: "That's great! How?".